conf/app.routes

```diff
@@ -1,15 +1,88 @@
+
+GET        /passport                                        controllers.PassportDetailsController.onPageLoad(mode: Mode = NormalMode)
+POST       /passport                                        controllers.PassportDetailsController.onSubmit(mode: Mode = NormalMode)
+GET        /change-passport                                 controllers.PassportDetailsController.onPageLoad(mode: Mode = CheckMode)
+POST       /change-passport                                 controllers.PassportDetailsController.onSubmit(mode: Mode = CheckMode)
+
+GET        /has-id-card                                     controllers.IdCardDetailsYesNoController.onPageLoad(mode: Mode = NormalMode)
+POST       /has-id-card                                     controllers.IdCardDetailsYesNoController.onSubmit(mode: Mode = NormalMode)
+GET        /change-has-id-card                              controllers.IdCardDetailsYesNoController.onPageLoad(mode: Mode = CheckMode)
+POST       /change-has-id-card                              controllers.IdCardDetailsYesNoController.onSubmit(mode: Mode = CheckMode)
+
+GET        /id-card                                         controllers.IdCardDetailsController.onPageLoad(mode: Mode = NormalMode)
+POST       /id-card                                         controllers.IdCardDetailsController.onSubmit(mode: Mode = NormalMode)
+GET        /change-id-card                                  controllers.IdCardDetailsController.onPageLoad(mode: Mode = CheckMode)
+POST       /change-id-card                                  controllers.IdCardDetailsController.onSubmit(mode: Mode = CheckMode)
+
+GET        /uk-address                                      controllers.UkAddressController.onPageLoad(mode: Mode = NormalMode)
+POST       /uk-address                                      controllers.UkAddressController.onSubmit(mode: Mode = NormalMode)
+GET        /change-uk-address                               controllers.UkAddressController.onPageLoad(mode: Mode = CheckMode)
+POST       /change-uk-address                               controllers.UkAddressController.onSubmit(mode: Mode = CheckMode)
+
+GET        /non-uk-address                                  controllers.NonUkAddressController.onPageLoad(mode: Mode = NormalMode)
+POST       /non-uk-address                                  controllers.NonUkAddressController.onSubmit(mode: Mode = NormalMode)
+GET        /change-non-uk-address                           controllers.NonUkAddressController.onPageLoad(mode: Mode = CheckMode)
+POST       /change-non-uk-address                           controllers.NonUkAddressController.onSubmit(mode: Mode = CheckMode)
+
+GET        /individual-added                                controllers.WhenIndividualAddedController.onPageLoad(mode: Mode = NormalMode)
+POST       /individual-added                                controllers.WhenIndividualAddedController.onSubmit(mode: Mode = NormalMode)
```

Comment:
We've conventionally called this route `when-added`
And as per my comment for the WhenIndividualAddedController it doesn't require a Mode to be passed to it as we know it's NormalMode only